### DISCUSSION TITLE
How to use `--security-opt` in Podman API

### DISCUSSION THREAD

**Discussion by ruspaul013** [containers/podman#24187] [Q&A] [answered]
Hello!

I am trying to add `security-opt` support in [nomad-driver-podman](https://github.com/hashicorp/nomad-driver-podman/issues/297) [[PR-link](https://github.com/hashicorp/nomad-driver-podman/pull/382)], but the API does not include `security_opt` as in docker. Is there anything I can do to map the values so that this flag can be used?

**Comment by Luap99** [comment, marked as answer]
They have their own option in the specgen format instead of one `security-opt` field that handles a bunch of different things
https://github.com/containers/podman/blob/35768aef2bb577d0cd183ee10b80bcba7f235af2/pkg/specgenutil/specgen.go#L700

So you pass `"no_new_privileges" : true` for example in the specgen json, seem like you PR already parses it.

**Reply by ruspaul013** [reply]
Thank you @Luap99. I will take a look.
